focus-ios/focus-ios-tests/XCUITest/BrowsingTest.swift

```diff
@@ -75,4 +75,64 @@ class BasicBrowsing: BaseTestCase {
         app.buttons["BrowserToolset.stopReloadButton"].tap()
         waitForValueContains(app.textFields["URLBar.urlText"], value: "mozilla")
     }
+
+    // Smoketest
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2587661
```

Comment:
TestRail is not updated correctly..
- Sub Test Suite should be `FullFuntional, Smoke & Sanity` (a test is always part of the full functional, then it could be smoke or not)
- Automation is Completed (not Untriaged)

Reply:
Added both `Functional` and `Smoke & Sanity` to the test. Automation is set to `completed`.